.github/workflows/pull-request.yml

```
@@ -14,16 +14,16 @@ jobs:
     runs-on: ubuntu-20.04
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@master
```

The main branch of `actions/checkout` is main, would you mind changing it?
```suggestion
      - uses: actions/checkout@main
```